repos/fdbt-site/src/utils/sessions.ts

```diff
@@ -229,6 +231,7 @@ export interface SessionAttributeTypes {
     [EDIT_CARNET_PROPERTIES_ERROR]: ErrorInfo[];
     [CAP_EXPIRY_ATTRIBUTE]: CapExpiry | ErrorInfo[];
     [CAP_START_ATTRIBUTE]: CapStartInfo | ErrorInfo[];
+    [CAP_PRICING_PER_DISTANCE_ATTRIBUTE]: { errors: ErrorInfo[]; capPricePerDistances: CapPricePerDistances[] };
```

Comment:
This means that the attribute will always have an errors array inside it, which seems needless. If you look at some of the others above, doing Data | ErrorInfo[] is one option, or doing Data | WithErrors<Data> is another option.